R/as_list.R

```diff
@@ -27,6 +27,11 @@ as_list <- function(x, ns = character(), ...) {
 }
 
 #' @export
+as_list.default <- function(x, ns = character(), ...) {
```

Comment:
Not much more work really, I just didn't know how we wanted to define that class, so using the default methods was a easy first draft.